makefile

```diff
@@ -1,12 +1,21 @@
 CXX = g++
-CXXFLAGS = -std=c++20 -Wall -g -MMD -MP -mavx2 -O3
+CXXFLAGS = -std=c++20 -Wall -MMD -MP -mavx2
 SRC_DIR = src
 OBJ_DIR = obj
 TARGET = pixelbasher
 
 SRCS = $(wildcard $(SRC_DIR)/*.cpp)
 OBJS = $(patsubst $(SRC_DIR)/%.cpp, $(OBJ_DIR)/%.o, $(SRCS))
 
+# Default target
+all: release
+
+debug: CXXFLAGS += -g -DDEBUG
```

Comment:
there's no need to define a `DEBUG` macro if you don't use it (the `NDEBUG` macro below is a different story, as it is observed by the `assert.h`/`cassert` standard header)